Shoukou Reshuffle Fix!

WhiteLightningTwister opened the pull request:
<!--
This is a semi-strict format, you can add/remove sections as needed but the order/format should be kept the same
Remove these comments before submitting
-->

# Description

Collected feedback which i had carefully read and sorted through to fix with this push with the recent shoukou map being finally pushed. I had made some minor mistakes which i corrected in this. Which should fix all the things i missed like wise others i had found.

They are all listed below for you to observe.


-->
# Station Overview image

<details><summary><h1>Media</h1></summary>
<p>

An overview image that shows some of the changes.

<img width="6208" height="4288" alt="Shoukou-0" src="https://github.com/user-attachments/assets/dcb372ec-ce81-446e-aa52-e07ba74155c2" />

<img width="776" height="725" alt="image" src="https://github.com/user-attachments/assets/80da978b-7142-49a4-82d3-4f997de4454f" />

<img width="829" height="690" alt="Screenshot 2025-09-11 172208" src="https://github.com/user-attachments/assets/69e47a80-96b7-406b-af5c-893e10a36dff" />

<img width="867" height="614" alt="Screenshot 2025-09-11 172152" src="https://github.com/user-attachments/assets/490916f8-980e-4251-90a5-071a88cde91c" />

</p>
</details>

# Changes

Additions:
- Added new EVAC pod areas.
- Added a new abandoned room by evac.

Tweaks;
- Fixed Crew monitor not working at round start.
- Fixed missing telecoms. Justice, perma and security should be available again.
- Moved EVAC pods around.
- Made sure court house had wiring this time.
- gave back epistemics stolen disposal shute.
- Placed more beacons for station navigation.
- Minor adjustments to epistemics artifact chamber.
- Added back round start artifacts to epistemics.
- Reconnected south east solars to the HV grid.
- Removed the plastisteel vend from perma.
- Adjusted the disposals end point to be more user friendly and less dis-functional.
- Renamed all the holopads correctly.
- decorated the station more.

# Changelog
:cl:
- add: Did a collective amount of fixing of minor errors, put things that were missing from the original back.

WhiteLightningTwister committed:
Shoukou Jobs Assigments

Added the justice department. Likewise tweaked and expanded existing jobs to reflect greater capacity. Due to the map changes.

WhiteLightningTwister committed:
Shoukou Mapreshuffle

A major rework of the map adding justice. Likewise making the map a SM station, moving and modifying almost every department to reflect new station parts and expansions, adding a proper station AI. Likewise reworking the the following departments.

Epistemics, Justice, Service, Cargo & Salvage, Security, Command.

WhiteLightningTwister committed:
Job Listing Error

There was a mistake where i left in the original lawyer so there was two. Fixed it.

WhiteLightningTwister committed:
Finished map changes

Adjusted some rooms and missing links including lights. Likewise fixing mis-placed map objects.

WhiteLightningTwister committed:
Map Wiring

Fixed missing LV

Dunrab committed:
Merge branch 'master' into ShoukouMapReshuffle

Dunrab committed:
Update shoukou.yml

remove duplicate justice roles

WhiteLightningTwister committed:
Merge branch 'Floof-Station:master' into ShoukouMapReshuffle

WhiteLightningTwister committed:
Adjusting Shoukou Shuffle Mistakes

Fixed wiring issues, likewise a mixture of complaints complied in the discord for shoukou that needed addressing. Such as no wiring in court and moving EVAC pods around, among other things.

WhiteLightningTwister committed:
Adjusted Some Minor mistakes

Placed disposals in epi, removed the plasteel vend from perma and replaced it with a crate of goods. Fixed disposal belt so its more functional. Placed Geiger counter & intercom on the SM.

WhiteLightningTwister committed:
Adjustments for shoukou

Adjusted a collection of broken airlocks, fixed APC names. Including beacons were adjusted.

WhiteLightningTwister committed:
More Minor edits

Fixed the cargo doors being all access. Added canisters to epi. Likewise made a grey tider den in the maints.

Dunrab commented:
@WhiteLightningTwister 

epi has no fax machine (only the mysta and mantis have private faxes)
did you mean to leave robotics on its own where epi used to be? feels a bit odd having robotics off on its own unchanged that far from epi

Checked the other things you said you changed and all looks good from what I can see.

Dunrab reviewed (changes requested):
Dorms, the lawyer office and the arcade are not connected to the an air sensor or an air alarm.

WhiteLightningTwister committed:
Adjustments requested by dunrab

Fixed air alarms in the law office, dorms including arcade area. Likewise placed a fax machine in epistemics. Fixed a holopad in the chaplains room, renamed part of dorms to upper dorms.

Dunrab reviewed (approved):
thank you for making those small changes, all checks passed good to go.
